ONSdigital/ras-rm-auth-service
# Fix warnings

# Motivation and Context
When running the unit tests, there were a number of warnings that occurred.  This PR fixes those issues.

# What has changed
Warnings fixed
A little bit of logging added

# How to test?
Create a user and login with it, then try and update it using response-operations-ui.  No functionality changed so it all should still work.

# Links
